functions.php

```diff
@@ -45,15 +49,42 @@ function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore) {
  * @param array $finals
  * @return string
  */
-function getTeamData(array $finals): string {
+function getTeamData(array $finals): string 
+{
     $team = '';
     foreach($finals as $final) {
         if ($final === []) {
             return 'Got data?';
         } else {
-            $team .= '<div class="winners-box">' .
-                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div></div>';
+            if($final['Delete'] == '0') {
```

Comment:
this is very inefficient. better to change your query in the `getData` function to only return data that hasn't been deleted

Reply:
WHERE statement included in getData function and 'if' statement removed from getTeamData function